FreeRTOS/FreeRTOS
# Update unit test for merging smp

Description
-----------
xYieldPending and xIdleTaskHandle are extended to arrays in FreeRTOS SMP. This PR update the unit test for single core FreeRTOS and SMP FreeRTOS.

>This PR should be merged before merging FreeRTOS SMP to main [PR](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/598)

Test Steps
-----------
Single core FreeRTOS test result can be found in github action.
FreeRTOS SMP test result can be found in this [PR](https://github.com/chinglee-iot/FreeRTOS/pull/3).

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
